contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -202,6 +196,46 @@ For example, the [`ci-kubernetes-node-kubelet`](https://github.com/kubernetes/te
 make test-e2e-node REMOTE=true FOCUS="\[NodeConformance\]" SKIP="\[Flaky\]|\[Serial\]"
 ```
 
+See [The Spec Runner](http://onsi.github.io/ginkgo/#the-spec-runner) in the Grinkgo documentation to learn more about how FOCUS and SKIP work.
```

Comment:
I think a link to this section would be more appropriate http://onsi.github.io/ginkgo/#focused-specs - it goes directly into skip and focus.

Reply:
May also be useful to cite or link to an example within Kubernetes.

Reply:
> I think a link to this section would be more appropriate http://onsi.github.io/ginkgo/#focused-specs - it goes directly into skip and focus.

I agree.

> May also be useful to cite or link to an example within Kubernetes.

Do you have a particular example in mind? I thought the example above was clear enough.